frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -157,13 +158,13 @@ function PlotlyFigure({
     )
     const storedValue = widgetMgr.getJsonValue(element)
 
-    if (storedValue !== undefined) {
+    if (storedValue !== undefined && storedValue !== "{}") {
```

Comment:
We probably want a comment mentioning that we store the serialized JSON element here, otherwise checking for a magic string seems pretty hacky

Reply:
Thanks. Will add a comment.